kubernetes-maven-plugin/doc/src/main/asciidoc/inc/build/_configuration.adoc

```diff
@@ -22,6 +22,13 @@ described in <<build-buildargs,Build Args>>
 https://docs.docker.com/engine/reference/api/docker_remote_api_v1.24/#build-image-from-a-dockerfile[Docker Remote API] and are restricted to simple options
 (e.g.: memory, shmsize). If you use the respective configuration options for build options natively supported by the build configuration (i.e. `noCache`, `cleanup=remove` for buildoption `forcerm=1` and `args` for build args) then these will override any corresponding options given here. The key-value syntax is the same as when defining environment variables or labels as described in <<misc-env,Setting Environment Variables and Labels>>.
 
+| *createImageOptions*
```

Comment:
This might be just my opinion. I think createImageOptions might be confusing from an end-user perspective. User also creates images of this project using build process. In my opinion it should be named `pullOptions`/ `pullImageOptions`, `importOptions` etc

Reply:
This is named according to the HTTP API exposed by Docker.

I agree it might be confusing, but just as `buildOptions`. Both these configurations are solely in the scope of the docker daemon and its HTTP API. IMHO making them as specific as possible, makes it harder for users to confuse them with something else.

However, I don't feel there is a better way to clear confusion. `pullOptions` or `importOptions` are confusing too i.e. what is the purpose of these options? Is this something generic? are these options applied when using S2I/JIB?

Considering a full example:
```xml
<build>
  <createImageOptions>
    <platform>linux</platform>
    <message>This is a docker commit message</message>
  </createImageOptions>
</build>
```

IMO, since this is something very specific to Docker, it's better to keep the name as specific as possible in order to prevent users having other expectations.

Reply:
ohk, let's keep it.